docs/topics/i18n/timezones.txt

```diff
@@ -198,12 +202,15 @@ Create a view that can set the current timezone::
 
     from django.shortcuts import redirect, render
 
+    # Prepare a list of common timezone choices you wish to offer.
+    common_timezones = [...]
+
     def set_timezone(request):
         if request.method == 'POST':
             request.session['django_timezone'] = request.POST['timezone']
```

Comment:
this view should validate the timezone is in `common_timezones`

Reply:
At the beginning of this section we have: 

> Here's an example that stores the current timezone in the session. (It skips
error handling entirely for the sake of simplicity.)

So I'm going to take error handling as out of scope (unless the example gets rewritten separately)